templates/Azure/Vm/approve-terms.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
```

Comment:
can you not use https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/marketplace_agreement

I have used it before and works OK.

Reply:
Done on the shared infra pipeline.
Can you review/approve this and we can just leave this task here.